src/main/resources/db/migration/V1__.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE message
+(
+    id              BIGINT AUTO_INCREMENT NOT NULL,
+    date            date                  NULL,
+    title           VARCHAR(255)          NULL,
+    message_body    VARCHAR(255)          NULL,
+    private_message BIT(1)                NOT NULL,
+    user_id         BIGINT                NULL,
+    CONSTRAINT pk_message PRIMARY KEY (id)
+);
+
+CREATE TABLE user
+(
+    id         BIGINT AUTO_INCREMENT NOT NULL,
+    user_name  VARCHAR(255)          NULL,
+    first_name VARCHAR(255)          NULL,
+    last_name  VARCHAR(255)          NULL,
+    email      VARCHAR(255)          NULL,
+    CONSTRAINT pk_user PRIMARY KEY (id)
+);
```

Comment:
There is no column for the image.

Reply:
Oops missed that completely, good catch :)